utils/partitions.js

```diff
@@ -35,7 +35,7 @@ exports.partition = function(dataset, testSetStart, testSetCount) {
  * @note code adapted from Heather Arthur:  https://github.com/harthur/classifier/blob/master/test/cross-validation/cross-validate.js
  */
 exports.partitions = function(dataset, numOfPartitions, callback) {
-	var shuffledDataset = _.shuffle(dataset);
+	// var shuffledDataset = _.shuffle(dataset);
```

Comment:
I used it for check
Now it's fixed


On Mon, Jan 5, 2015 at 2:19 PM, Erel Segal-Halevi <notifications@github.com>
wrote:

> In utils/partitions.js
> <https://github.com/erelsgl/limdu/pull/23#discussion-diff-22458188>:
>
> > @@ -35,7 +35,7 @@ exports.partition = function(dataset, testSetStart, testSetCount) {
> >   * @note code adapted from Heather Arthur:  https://github.com/harthur/classifier/blob/master/test/cross-validation/cross-validate.js
> >   */
> >  exports.partitions = function(dataset, numOfPartitions, callback) {
> > -	var shuffledDataset = _.shuffle(dataset);
> > +	// var shuffledDataset = _.shuffle(dataset);
>
> Why did you remove the shuffling?
>
> In cross-validation, it is common to shuffle the data in order to make
> sure that there is no bias in the partition.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/erelsgl/limdu/pull/23/files#r22458188>.
>